packages-ts/gauntlet-cosmos-contracts/networks/.env.local

```diff
@@ -1,4 +1,4 @@
-NODE_URL=http://localhost:26657
```

Comment:
for some reason, wasmd does not support ipv6 so you have to specify the ipv4 address